x-pack/platform/plugins/shared/ml/public/application/timeseriesexplorer/components/timeseries_chart/timeseries_chart.js

```diff
@@ -87,7 +91,13 @@ const ZOOM_INTERVAL_OPTIONS = [
 const anomalyColorScale = d3.scale
   .threshold()
   .domain([3, 25, 50, 75, 100])
-  .range(['#d2e9f7', '#8bc8fb', '#ffdd00', '#ff7e00', '#fe5050']);
+  .range([
+    ML_SEVERITY_COLORS.LOW,
+    ML_SEVERITY_COLORS.WARNING,
+    ML_SEVERITY_COLORS.MINOR,
```

Comment:
`MINOR` and `MAJOR` are slightly different here (than #ffdd00 and #ff7e00), but I assume this is intentional?

Reply:
Again thanks for your attention to detail! Discussed with Pete and we can't remember if this was intentional or an oversight originally, so we'll move forward with the consolidation to make it the same color.